datafusion/sqllogictest/test_files/arrow_typeof.slt

```diff
@@ -424,7 +424,7 @@ select arrow_cast([1, 2, 3], 'FixedSizeList(3, Int64)');
 [1, 2, 3]
 
 # Tests for Utf8View
-query ?T
+query TT
```

Comment:
It was after I made sqllogictest treat utf8view as Text that I found this way, by using a `?`, to represent unknown column. I wonder if my change in normalize.rs is correct.

Reply:
I believe it is correct -- I actually have a PR to do the same change in https://github.com/apache/datafusion/pull/12033 but it hasn't merged yet. It is good to do it in this PR instead. Thank you